manifests/server/grant.pp

```diff
@@ -302,18 +327,63 @@
       $_togrant_object = join($_object_name, '"."')
       # Never put double quotes into has_*_privilege function
       $_granted_object = join($_object_name, '.')
+      $_schema = $_object_name[0]
+      $_relation = $_object_name[1]
     }
     default: {
       $_granted_object = $_object_name
       $_togrant_object = $_object_name
+      $_schema = $_object_name
+      $_relation = '%'
```

Comment:
disclaimer: i don't know anything about postgres :)

What does relation = '%' mean? all relations?

Feel free to be very terse here. Mostly just curious, and my google-fu couldn't find the answer.

Reply:
% is a wildcard character in ANSI SQL described here: https://www.w3schools.com/sql/sql_wildcards.asp. Postgres, like its brethren, is a dialect of ANSI SQL.

I'm still mid-testing to verify if equivalence will work or a LIKE query will be required. If the latter is required, that will change here.